serialization/api.go

```diff
@@ -28,10 +35,10 @@ type IdentifiedDataSerializableFactory interface {
 // Each IdentifiedDataSerializable is created by a registered IdentifiedDataSerializableFactory.
 type IdentifiedDataSerializable interface {
 	// FactoryID returns IdentifiedDataSerializableFactory factory ID for this struct.
-	FactoryID() (factoryID int32)
+	FactoryID() int32
```

Comment:
Can this be a breaking change? I mean, can there be users relying on named return values (or whatever they are called in Go)?

Reply:
No, the name of the return variable has no meaning to the caller, and the caller cannot access that.